apps/storybook/src/LineVis.stories.tsx

```diff
@@ -33,24 +31,39 @@ const errorsArray = ndarray(
   dataArray.shape
 );
 
-const Template: Story<LineVisProps> = (args) => <LineVis {...args} />;
+const Template: Story<LineVisProps> = (args) => {
+  const {
+    dataArray,
+    domain: storyDomain,
+    scaleType,
+    errorsArray,
+    auxArrays,
+  } = args;
+
+  // If story doesn't provide `domain`, compute it automatically
+  const domain =
+    storyDomain ||
+    getCombinedDomain([
+      getDomain(dataArray, scaleType, errorsArray),
+      ...(auxArrays ? getDomains(auxArrays, scaleType) : []),
+    ]);
```

Comment:
If the user changes the scale type with the controls, the domains we compute ahead of time for the stories may no longer be valid or relevant. Also, computing the domain for so many stories was starting to get out of hand. I still pass a `domain` to the stories where it's the basic domain computed with `getDomain(dataArray)` -- I thought this would still be important to do for documentation purposes, but when we start passing `errorsArray`, `auxiliaryArrays` or scales, this block of code takes over.